.php_cs

```diff
@@ -0,0 +1,20 @@
+<?php
+// see https://github.com/FriendsOfPHP/PHP-CS-Fixer
+
+$finder = PhpCsFixer\Finder::create()
+    ->exclude('vendor')
+    ->in([__DIR__])
+;
+
+return PhpCsFixer\Config::create()
+    ->setRiskyAllowed(true)
+    ->setRules([
+        '@PHP71Migration:risky' => true,
```

Comment:
What about adding the `@Symfony` rules?

Reply:
I considered that option, but then I kept that rules out, since after all this an agnostic library (hoping it's used outside Symfony too)